roxie/udplib/udptrs.cpp

```diff
@@ -384,11 +386,12 @@ class UdpReceiverEntry : public IUdpReceiverEntry
     void resendRequestToSend()
     {
         // This is called from the timeout thread when a previously-send request has had no response
+        okToSendTimeouts++;
```

Comment:
I think this (and timeouts) is only incremented on a single thread.  If so using fastInc() would avoid the atomic operation.  Strictly speaking it should stay atomic because it is read from a different thread.